game-core/src/main/java/games/strategy/engine/framework/startup/ui/panels/main/game/selector/GameSelectorModel.java

```diff
@@ -218,6 +218,8 @@ private static boolean gameUriExistsOnFileSystem(final String gameUri) {
     } catch (final IllegalArgumentException ignored) {
       // thrown if the URI is invalid (EG: missing URI scheme)
       return false;
+    } catch (final FileSystemNotFoundException ignored) {
+      // the file system will be created inside of getDefaultGameRealPath
```

Comment:
Forward referencing another method in a comment is an odd form of coupling.

I think the fix in: https://github.com/triplea-game/triplea/pull/8066 is a bit more direct.

I will admit I am disappointed that Path.of breaks if it references a path inside of a zip file via URI.

Reply:
Yes, that is better.  I'll change this PR to just include the new comments.